RealmSwift/SwiftUI.swift

```diff
@@ -651,47 +645,21 @@ extension Projection: _ObservedResultsValue { }
             }
 
             value = results.sectioned(sortDescriptors: sortDescriptors, sectionBlock)
-
-            setupHasRun = true
```

Comment:
Removing this and move it to `didSet` results in this function been called endlessly from `wrappedValue`. This also applies to `@ObservedSectionedResults`. I tested this on a project and restoring this and removing it from didSet solves the issue

Reply:
Good catch; I accidentally made it so that the only place `setupHasRun = true` was set was inside a `if setupHasRun { }` block. I don't see an obvious way to catch this from a test because I think it still always gave correct results and was just a lot slower than it should be.